docs/DeveloperGuide.md

```diff
@@ -333,6 +333,34 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 *{More to be added}*
 
+
+**Use case: Add a meeting**
+
+**MSS**
+
+1.  User requests to list meetings
+2.  AddresSoc shows a list of meetings
+3.  User requests to add a meeting in the list
+4.  AddresSoc deletes the person
+    
+    Use case ends.
+
+**Extensions**
+
+* 3a. The tags are invalid.
```

Comment:
Since we use the term `tags` as a parameter in `add contact`, maybe we could use another word here like parameters or arguments provided, etc. Same for line 356